transportable-udfs-trino-plugin/build.gradle

```diff
@@ -0,0 +1,33 @@
+apply plugin: 'java'
+apply plugin: 'distribution'
+
+java {
+  toolchain.languageVersion.set(JavaLanguageVersion.of(17))
+}
+
+dependencies {
+  implementation project(':transportable-udfs-api')
+  implementation project(":transportable-udfs-trino")
+  implementation(group:'io.trino', name: 'trino-main', version: project.ext.'trino-version') {
```

Comment:
this should be a `testImplementation` dependency? we should ONLY be depending on `trino-spi` IIUC, that was the point of the 406 changes -- to put all of the UDF interfaces we need into the SPI

Reply:
Done.